### PR TITLE
Defect fixing for case discussion and problem statement

### PR DESCRIPTION
**Pull request by renujhamtani**
@pbathia @ardasgup Please review and merge.

### CODE REVIEW

File: app/ascension/controllers/caseDiscussion.js [hunk 1]
```diff
@@ -70,6 +70,10 @@ angular.module('RedhatAccess.ascension').controller('CaseDiscussion', [
                 filter: 'bugzillas'
             },
             {
+                name: gettextCatalog.getString('View Chat Transcripts'),
+                filter: 'bugzillas'
```
Comment (pbathia) [inline app/ascension/controllers/caseDiscussion.js]: this filter you will change to chat transcript, when you add it right ?

**Comment by pbathia** [comment]
Rest all looks good.

File: app/ascension/controllers/caseDiscussion.js (hunk 1 above)
Comment (renujhamtani) [inline app/ascension/controllers/caseDiscussion.js]: This was already implemented(chat transcripts) one...spenser forgot to add this...

**Comment by ardasgup** [comment]
Reviewed!

**Comment by renujhamtani** [comment]
@pbathia I have incorporated ur review comments and also fixed the defect for TAM, internal status and NEP. Please review

**Comment by pbathia** [comment]
Looks good.

**Comment by ardasgup** [comment]
Reviewed again :)

**Pull request merged**